libraries/protocol/steem_operations.cpp

```diff
@@ -564,20 +564,32 @@ namespace steem { namespace protocol {
    {
       validate_account_name( owner );
 
-      FC_ASSERT( amount_to_sell.symbol == exchange_rate.base.symbol, "Sell asset must be the base of the price" );
       exchange_rate.validate();
 
-      FC_ASSERT(  ( is_asset_type( amount_to_sell, STEEM_SYMBOL ) && is_asset_type( exchange_rate.quote, SBD_SYMBOL ) )
-               || ( is_asset_type( amount_to_sell, SBD_SYMBOL ) && is_asset_type( exchange_rate.quote, STEEM_SYMBOL ) )
-               || (
-                     amount_to_sell.symbol.space() == asset_symbol_type::smt_nai_space
-                     && is_asset_type( exchange_rate.quote, STEEM_SYMBOL )
-                  )
-               || (
-                     is_asset_type( amount_to_sell, STEEM_SYMBOL )
-                     && exchange_rate.quote.symbol.space() == asset_symbol_type::smt_nai_space
-                  ),
-               "Limit order must be for the STEEM:SBD or SMT:(STEEM/SBD) market" );
+      FC_ASSERT( amount_to_sell.symbol.is_vesting() == false, "Cannot create a limit order for vesting types." );
```

Comment:
These additions might be able to be skipped based on the changed you made in the evaluator...

Reply:
I tested locally and removing this entire block does not break the tests.

Reply:
I tested this locally, and it can be removed without breaking the tests.